integration/test/draining/draining_test.go

```diff
@@ -212,15 +264,21 @@ func Test_Draining(t *testing.T) {
 			t.Fatalf("expected %#v got %#v", nil, err)
 		}
 
+		newLogger.Log("level", "debug", "message", "scaled down tenant cluster worker")
+	}
+
+	{
 		newLogger.Log("level", "debug", "message", "verifying e2e-app availability 10 more seconds")
 		time.Sleep(10 * time.Second)
-		close(done)
+		newLogger.Log("level", "debug", "message", "verified e2e-app availability 10 more seconds")
 	}
 
+	close(done)
+
 	{
 		newLogger.Log("level", "debug", "message", "validating test data")
 
-		acceptable := float64(20)
+		acceptable := float64(0)
```

Comment:
Does this succeed reliably right now?

Reply:
I talked about this in standup. Yes it does.

Reply:
If it degrades we can easily bump the threshold again.